README.md

```diff
@@ -23,3 +23,13 @@ jekyll serve
 (or `jekyll serve --destination TMPDIR`) if you want to use a different destination directory.
 
 And then browse to `http://127.0.0.1:4000`
+
+## Running jekyll in docker
+It is also possible to serve the site locally from a docker container.
+To do this, run this command from the repository directory:
+```
+docker run -v $PWD/:/srv/jekyll -e JEKYLL_VERSION=3.8 -p 4000:4000 jekyll/jekyll:3.8 jekyll serve
+```
+* `-v $PWD/:/srv/jekyll`: Mount the current working directory to `/srv/jekyll` inside the docker 
+container. This is where the jekyll docker image expects the source files to be.
+*  
```

Comment:
```suggestion

```

Reply:
Remove the empty bullet point.

Reply:
That worked for me, thank you for this addition. I proposed removing the empty bullet point at the end for the README. Otherwise, it's ready to merge.

Reply:
Removed it